docs/Gemfile.lock

```diff
@@ -262,7 +262,7 @@ DEPENDENCIES
   github-pages
 
 RUBY VERSION
-   ruby 2.7.1p83
+   ruby 2.7.0p0
```

Comment:
I'm not sure about adding the `Gemfile.lock` to this project. Since you're downgrading the version from 2..7.1 to 2.7.0. I'm not sure about the conventions we follow for pushing `Gemfile.lock`, Anyway, if we allow to add Gemfile.lock, I suggest you to use the same ruby version the project is having in this case: ruby 2.7.1p83 an bundler 2.1.4.

Reply:
Agreed @jesuslerma since in my system which uses WSL2, I used Ruby 2.7.0, that's why the version is different. I tried upgrading to 2.7.1 but it's not available for me it seems. Anyway, I'll make that change and come back!

Reply:
Oh and the bundler..... It's still upgraded even if I downgrade the version, you can check that though.